lib/src/impl_js/impl_js.utils.dart

```diff
@@ -99,6 +99,13 @@ Object _translateDomException(
       '"${e.name}", message: $message');
 }
 
+Object _translateJavaScriptException(Error e) {
+  // dart2wasm throws _JavaScriptError, but _JavaScriptError is not exposed.
+  final errorMessage = e.toString();
+  final message = 'browser threw "$errorMessage"';
+  return ArgumentError(message);
```

Comment:
I don't know what we should throw here. This could be `UnsupportedError`, `ArgumentError` or even `OperationError`.

We might need to throw a new `UnknownError` that contains a message explaining how this version of `package:webcrypto` cannot distinguish between the different kinds of errors thrown by Javascript.

Possibly with a link to an issue in our issue tracker on this repository.